cmd/core-command/res/configuration.toml

```diff
@@ -1,11 +1,5 @@
 [Writable]
-LogLevel = "INFO"
   [Writable.InsecureSecrets]
-    [Writable.InsecureSecrets.DB]
-    path = "redisdb"
-      [Writable.InsecureSecrets.DB.Secrets]
-      username = ""
-      password = ""
     [Writable.InsecureSecrets.mqtt]
```

Comment:
This needs to moved to common all-service. Was missing from some services config

Reply:
this is in common config

Reply:
The one for `redisdb` is in common, but not the one for `mqtt`